gordo_components/data_provider/iroc_reader.py

```diff
@@ -59,14 +59,12 @@ def load_series(
             map(lambda tag: IrocReader.base_path_from_asset(tag.asset), tag_list)
         )
         if len(set(base_paths_from_assets)) > 1:
-            logger.warning(
-                "Iroc reader found more than one asset from the tag list provided, returning none."
+            raise ValueError(
+                "Iroc reader found more than one asset from the tag list provided"
             )
             return
```

Comment:
The return can never be reached, remove it

Reply:
Done!